### PR TITLE
Add JavaScript questions.

### PR DESCRIPTION
**Pull request by populationtwo**
Added a few JavaScript questions. Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,11 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
+* What is a Callback functions and explain when do you use it?
+* What is Variable Scope in JavaScript? What's the difference between Local Variables  and Global  Variables?
+* Explain Object Oriented Programming in Javascript.
+
+
```
Comment (arthurvr) [inline README.md]: remove these blank lines

File: README.md [hunk 2]
```diff
@@ -136,6 +136,11 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
+* What is a Callback functions and explain when do you use it?
+* What is Variable Scope in JavaScript? What's the difference between Local Variables  and Global  Variables?
+* Explain Object Oriented Programming in Javascript.
```
Comment (arthurvr) [inline README.md]: I prefer more to the point questions then this really general kind of questions.

File: README.md [hunk 3]
```diff
@@ -136,6 +136,11 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
+* What is a Callback functions and explain when do you use it?
```
Comment (arthurvr) [inline README.md]: > Callback functions

File: README.md [hunk 4]
```diff
@@ -136,6 +136,11 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
...
+* What is Variable Scope in JavaScript? What's the difference between Local Variables  and Global  Variables?
```
Comment (arthurvr) [inline README.md]: Why not to take this question a little further? Something like

> What are the differences between variables created using `let`, `var` or `const`?

File: README.md (hunk 3 above)
Comment (populationtwo) [inline README.md]: Yes. I think the answer should be like what's explained in this article: http://javascriptissexy.com/understand-javascript-callback-functions-and-use-them/

File: README.md (hunk 2 above)
Comment (populationtwo) [inline README.md]: I agree with you. I just want to share the exact questions I have been asked on my recent interviews.

**Comment by darcyclarke** [comment]
@populationtwo  can you rebase? We've made some merges and they've unfortunately made this out-of-date.

**Comment by arthurvr** [comment]
ping @populationtwo

**Comment by arthurvr** [comment]
@populationtwo ping ;)

**Comment by richgilbank** [comment]
@populationtwo I'm going to close this for now as there haven't been any updates here in a while. Feel free to reopen if you get a chance to rebase.

**Comment by populationtwo** [comment]
@arthurvr @darcyclarke @richgilbank 
I'm sorry for the delayed response. I just read the notification today.

Thanks

**Pull request closed without merging**